src/index.js

```diff
@@ -29,6 +29,15 @@ const client = new Discord.Client({ partials: ["MESSAGE", "REACTION"] });
 
 const users = {}; // Message counter object for users
 const chnlMsgs = {}; // Separate message counter object unrelated to users
+const isMessageIgnored = (message) => {
+	if (!message.guild) return true; // Ignores DMs
+	if (message.author.bot) {
+		if (message.type === "PINS_ADD") message.delete();
+		return true; // Ignores messages from bots
+	}
+	if (message.type === "PINS_ADD") return true; // Ignores PIN messages
+	return false;
+} //Ignores messages
```

Comment:
I think this function is named well enough that this comment isn't necessary, it can be removed 
```suggestion
}
```